lms/templates/discussion/_underscore_templates.html

```diff
@@ -398,16 +398,16 @@ <h1 class="home-title">${course.display_name_with_default}</h1>
                     <option value="${'<%= opt.value %>'}" ${'<% if (opt.selected) { %>selected<% } %>'}>${'<%- opt.text %>'}</option>
                     ${'<% }); %>'}
                  </select>
-            </label><div class="field-help">
+            </label><div class="field-help" id="field_help_visible_to">
                 ${_("Discussion admins, moderators, and TAs can make their posts visible to all students or specify a single cohort.")}
             </div>
         </div>
         ${'<% } %>'}
         <div class="post-field">
-            <label class="field-label">
+            <label class="field-label" aria-describedby="field_help_title">
```

Comment:
hey @clrux, can you move this to the input element below?  It needs to be on a focusable element to work.